UI: Update Youtube get stream key URL

mv2731 opened the pull request:
The old URL is going to be deprecated soon, and this is why we replaced it
with a new one.

mv2731 committed:
UI: Update Youtube get stream key URL

The old URL is going to be deprecated soon, and this is why we replaced
with a new one.

ushadow commented on UI/window-basic-auto-config.cpp:
```diff
@@ -567,7 +567,7 @@ void AutoConfigStreamPage::UpdateKeyLink()
 			"https://www.twitch.tv/broadcast/dashboard/streamkey";
 	} else if (serviceName == "YouTube / YouTube Gaming") {
-		streamKeyLink = "https://www.youtube.com/live_dashboard";
+		streamKeyLink = "https://youtube.com/livestreaming";
```
Add www before youtube.com: https://www.youtube.com/livestreaming

ushadow commented on UI/window-basic-settings-stream.cpp:
```diff
@@ -242,7 +242,7 @@ void OBSBasicSettings::UpdateKeyLink()
 			"https://www.twitch.tv/broadcast/dashboard/streamkey";
 	} else if (serviceName == "YouTube / YouTube Gaming") {
-		streamKeyLink = "https://www.youtube.com/live_dashboard";
+		streamKeyLink = "https://youtube.com/livestreaming";
```
Same here, add www.

mv2731 committed:
UI: Added www to Youtube stream key URL

Added www to be beginning of the stream key URL we are using to replace
the deprecated live_dashboard Youtube stream key link.
